configuration/ide/eclipse/launchers/JMC-RCP.launch

```diff
@@ -31,73 +31,92 @@
    WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
    WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 -->
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>

```

Comment:
The presence of this line is preventing me from importing and using the launcher. Otherwise it seems to be working okay.